docs/source/roadmap.rst

```diff
@@ -16,34 +16,30 @@ contributions. Here's our plans for the next two releases.
     the feature. Pull Requests are open to anyone.
 
 
-3.1
----
-
-* **Ubuntu:** GA Support Ubuntu 18.04, with Python 3.6
-* **MongoDB:** Support MongoDB 4.0 (required for Ubuntu 18.04).
-* **Ubuntu:** Drop Ubuntu 14.04 support.
```

Comment:
Just so you're aware, we didn't really deprecate U14 support in `v3.1`, see https://github.com/StackStorm/st2/issues/4747. To avoid moving it under the "History" section.

Reply:
I'm going to communicate in the roadmap that we drop support of u14 in 3.1 but packages are still available.  I also added an entry in v3.3 to stop producing anymore u14 packages.